### DISCUSSION TITLE
Vercel does not support node `v20.13.0`

### DISCUSSION THREAD

**Discussion by NotArpit** [vercel/community#6793] [Help] [answered]
### Summary

Using the following engines constraint in `package.json`:
```
  "engines": {
    "node": ">=20.13.0",
  }
```

Yields the following error when deploying on Vercel.

```
Your Node version is incompatible with "/vercel/path0".
--
Expected version: >=20.13.0
Got: v20.12.2
```

It seems setting the runtime to `20.x` does not pull in the latest minor version?

### Steps to Reproduce

1. Add the following engines constraint to your `package.json` for your Next.js project.
```
"node": ">=20.13.0",
```
2. Deploy onto Vercel and observe the above error.

**Comment by swarnava** [comment, marked as answer]
That's correct. We don't support Node 20.13.0 yet and advise not to pin minor or patch version. Our runtime automatically upgrade to latest version when they are available by our upstream provider.

The recommendation is to use `20.x` instead so your build ensure always run on latest Node version when they are available.

**Reply by NotArpit** [reply]
I see - thanks for answering.
